docs/known-limitations/known-limitations.md

```diff
@@ -104,6 +104,9 @@ EVPN routes. This leads to blackholing of traffic for endpoints connected to the
 isolated leaf, as the rest of the fabric no longer has reachability information for
 those endpoints, even though the MCLAG peering session is up.
 
+!!! note
+    This issue was fixed in release v25.2.
```

Comment:
```suggestion
    This issue is fixed in release v25.2.
```

Or use present tense and merge now? (I don't mind either way)

Reply:
I guess `is` sounds a little better, but I'm not a native speaker and it probably doesn't matter :)

Reply:
`is` makes it sound less like “a thing we've done and released already” - in my opinion. That's all :)

Reply:
it's actually should be `25.02` (no `v` in the beginning and `0` before minor version) - https://docs.hedgehog.cloud/latest/reference/releases/